11-spatial-cv.Rmd

```diff
@@ -80,7 +80,7 @@ There are 175 landslide points and 1360 non-landslide, as shown by `summary(land
 The 1360 non-landslide points were sampled randomly from the study area, with the restriction that they must fall outside a small buffer around the landslide polygons.
 
 To make the number of landslide and non-landslide points balanced, let us sample 175 from the 1360 non-landslide points.^[The `landslides` dataset has been used in classes and summer schools.
-To show how predictive performance of different algorithms changes with an unbalanced and highly spatially autocorrelated response variable, 1360 non-landslide points were randomly selected, i.e. much more absences than presences.
+To show how predictive performance of different algorithms changes with an unbalanced and highly spatially autocorrelated response variable, 1360 non-landslide points were randomly selected, i.e., much more absences than presences.
```

Comment:
Minor comment, I think this line is better (grammatically correct) as follows:

```
To show how predictive performance of different algorithms changes with an unbalanced and highly spatially autocorrelated response variable, 1360 non-landslide points were randomly selected, i.e., many more absences than presences.
```

The use of 'much' incorrectly when referring to countable quantities is common, even among native speakers: https://www.gymglish.com/en/english-grammar/much-vs-many

'absences' is clearly a countable entity.

Reply:
This was well-spotted. Thanks, wouldn't have noticed that!